migrations/env.py

```diff
@@ -55,7 +55,31 @@
 """
 )
 
-register_entities([va_profile_opt_in_out])
+#######################################################################
+# Define and register the stored procedures for VA Profile integration.
+#######################################################################
+
+va_profile_delete_cron_job = PGFunction(
+  schema="public",
+  signature="va_profile_delete_cron_job()",
+  definition="""\
+RETURNS boolean AS
+$$
+DECLARE number_of_changed_records Int;
+BEGIN
+DELETE
+FROM va_profile_local_cache
+WHERE allowed = False 
+AND EXTRACT(DAY FROM NOW()::timestamp - source_datetime) > 1;
```

Comment:
I'm not sure this is correct.  If the query runs at 5 minutes after midnight, it will delete any opt-out older than 5 minutes.  It also looks like you're comparing a date to a datetime.  Did you manually test this on a container?